resource-service/src/main/resources/application-static.yaml

```diff
@@ -41,4 +41,4 @@ akka:
 # Connection detail for SimpleResourceService resources
 web:
   client:
-    data-service: http://localhost:8082
+    data-service: localhost:8082
```

Comment:
Change aside, why does the resource-service need to have a URL for the data-service?

Reply:
This is needed for the `ConnectionDetail` that's added to the resource object.